### PR TITLE
Added documentation how to get a foursquare access token

### PR DESCRIPTION
**Pull request by simonszu**
**Description:**
I struggled a bit with adding the foursquare component to my Home Assistant Setup, since it was not clear how to get the `access_token`. After a bit of googling i succeeded, and decided it would be a good idea to add a step-by-step description to the foursquare component documentation of how to get the access_token. 

No according pull request to the main Home Assistant source, since it is only more documentation for an existing component.

### CODE REVIEW

File: source/_components/foursquare.markdown [hunk 1]
```diff
@@ -28,6 +28,23 @@ Configuration variables:
...
 - **push_secret** (*Required*): The push secret that Foursquare provides to you in the app dashboard.
 
+#### Getting the access token ####
+
+After you have registered your APP on your [My Apps Page](https://foursquare.com/developers/apps) you get a `CLIENT_ID` and you have specified a
+`REDIRECT_URL` which can be any URL you like, but since it will get your access token via a HTTP GET request, it should be a URL which will ignore the `access_token` HTTP GET variable. 
+Visit the following URL in your browser:
+
+```
+https://foursquare.com/oauth2/authenticate
```
Comment (Landrash) [inline source/_components/foursquare.markdown]: Should these be on separate lines? A tad easier to read but might be harder for a new reader.

Comment (simonszu) [inline source/_components/foursquare.markdown]: Yes, you're right. I just copy&pasted this URL from the Foursquare API documentation.

**Comment by simonszu** [comment]
I have made the requested changes. Looks there were some heavier updates in upstream after my initial pull request, though. ;)

**Comment by simonszu** [comment]
I have fixed my commit, please look at https://github.com/home-assistant/home-assistant.github.io/pull/1492. Will close this now.

**Comment by Landrash** [comment]
@simonszu Been there done that. I'll review #1492

**Pull request closed without merging**